### ISSUE TITLE
Ex2.3.3 answer might be floor(N/2)

### ISSUE THREAD

**Issue by ajfg93** [reneargento/algorithms-sedgewick-wayne#30]
https://stackoverflow.com/questions/43263249/number-of-largest-element-exchanges-for-quicksort

I don't understand it very well but it seems reasonable.

**Comment by reneargento** [comment]
You are right, in the quicksort version used in the book, in which the first element is always selected as the pivot, the maximum number of times that the largest item can be exchanged is floor (N / 2).
I updated the exercise here: https://github.com/reneargento/algorithms-sedgewick-wayne/commit/7d4cb8ba177e627fa83475f15729088e5e6d700d
Thanks!
